tests/functional/test_source_metadata.py::TestInstanceMetadata::test_instance_metadata error on Focal

## Description

tests/functional/test_source_metadata.py::TestInstanceMetadata::test_instance_metadata is failing on Focal

## Steps to Reproduce

- `BASE_OS=focal securedrop/bin/dev-shell bin/run-test -s -v tests/functional/test_source_metadata.py`

## Expected Behavior

All tests should pass

## Actual Behavior

```
___________________________________________________________________________________ TestInstanceMetadata.test_instance_metadata ____________________________________________________________________________________
                                                                                                                                                                                                                    
self = <tests.functional.test_source_metadata.TestInstanceMetadata object at 0x7fe0f92878e0>                                                                                                                        
                                                                                                                                                                                                                    
    def test_instance_metadata(self):                                                                                                                                                                               
>       self._source_checks_instance_metadata()                                                                                                                                                                     
                                                                                                                                                                                                                    
/root/code/securedrop/securedrop/tests/functional/test_source_metadata.py:10:                                                                                                                                       
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
                                                                                                                                                                                                                    
self = <tests.functional.test_source_metadata.TestInstanceMetadata object at 0x7fe0f92878e0>                                                                                                                        
                                                                                                                                                                                                                    
    def _source_checks_instance_metadata(self):                                                                                                                                                                     
        self.driver.get(self.source_location + "/metadata")                                                                                                                                                         
        j = json.loads(self.driver.find_element_by_tag_name("body").text)                                                                                                                                           
>       assert j["server_os"] == "16.04"                                                                                                                                                                            
E       AssertionError            
```

## Comments

Suggestions to fix, any other relevant information.

--- a/securedrop/tests/functional/source_navigation_steps.py
+++ b/securedrop/tests/functional/source_navigation_steps.py
@@ -29,7 +29,7 @@ class SourceNavigationStepsMixin:
     def _source_checks_instance_metadata(self):
         self.driver.get(self.source_location + "/metadata")
         j = json.loads(self.driver.find_element_by_tag_name("body").text)
-        assert j["server_os"] == "16.04"
+        assert j["server_os"] in ["16.04", "20.04"]
         assert j["sd_version"] == self.source_app.jinja_env.globals["version"]
         assert j["gpg_fpr"] != ""
 

